src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>Should we mention that to maxime the objective we can use the field "scaler=-1.0"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Corrected in a6b9f26f

Reply:
<p>Actually, this change did not occurred. The mentioned commit does not modify this file, and I can't see any mention to the scaler parameter.</p>

Reply:
Indeed ! I have no idea why I resolved the conversation. Anyway, I fixed it now (91dd1f08), mentionning the way of doing it in OpenMDAO and in FAST-OAD